### DISCUSSION TITLE
Using this interceptor with Apisauce

### DISCUSSION THREAD

**Discussion by AlaricGit** [arthurfiorette/axios-cache-interceptor#642] [General]
Can you please provide an example to use this interceptor with Apisauce? 

`  api: ApisauceInstance;`

```
    api = apisauce.create({
      baseURL,
      headers,
      timeout,
    });
```

**Comment by AlaricGit** [comment]
BTW, Apisauce is a wrapper of Axios 
https://github.com/infinitered/apisauce

**Reply by arthurfiorette** [reply]
I don've never used `apisauce` before. It seems to be an axios wrapper, so you can just use the `setupCache` as normal. Make sure to pass in the axios instance in the parameter, not the `apisauce` wrapper.

**Reply by arthurfiorette** [reply]
You can follow the steps in their readme.

https://github.com/infinitered/apisauce#mocking-with-axios-mock-adapter-or-other-libraries

**Comment by AlaricGit** [comment]
The request and response parameters are different. it seems it is not easy to convert them to each other. Anyway, if you don't have experience with Apisauce no need to talk about more. 
Thanks a lot for your answer
